src/plugins/dashboard/public/dashboard_app/dashboard_app.test.tsx

```diff
@@ -43,10 +51,11 @@ describe('Dashboard App', () => {
       />
     );
     expect(expandedPanelIdSpy).toHaveBeenCalledTimes(1);
-    expect(historySpy).toHaveBeenCalledTimes(1);
+    expect(historySpy).toHaveBeenCalledTimes(2);
```

Comment:
Can we clear `historySpy` before each test so that this count can be `1` instead of `2`? This would be best done in a `beforeEach` :)

Reply:
[4234e10](https://github.com/elastic/kibana/pull/190086/commits/4234e10a3f4bbc2e935ca01f6c633c39a6606b7a) thank you!!!